qfi_opt/examples/benchmark_opt_methods.py

```diff
@@ -8,15 +8,26 @@
 from qfi_opt import spin_models
 from qfi_opt.examples.calculate_qfi import compute_QFI
 
-sys.path.append("/home/jlarson/research/poptus/orbit/py")
-sys.path.append("/home/jlarson/research/poptus/minq/py/minq5/")
-sys.path.append("/home/jlarson/research/poptus/IBCDFO/pounders/py")
-from general_h_funs import identity_combine as combinemodels
-from orbit4py import ORBIT2
-from pounders import pounders
+try:
+    from ibcdfo.pounders import pounders
+    from ibcdfo.pounders.general_h_funs import identity_combine as combinemodels
+except:
+    sys.exit("Please 'pip install ibcdfo'")
+
+try:
+    sys.path.append("../../../minq/py/minq5/")  # Needed by pounders, but not pip installable
+    from minqsw import minqsw
+except:
+    sys.exit("Make sure the MINQ [https://github.com/POptUS/minq] is on your path")
+
+# sys.path.append("../orbit/py")
+# from orbit4py import ORBIT2
 
 
 def nlopt_wrapper(x, grad, obj, obj_params):
+    # Note that this objective function creates/updates a database of 
+    # simulations inputs/outputs. For large databases (or fast simulations), the
+    # database lookup can be more expensive than performing the simulation.
```

Comment:
(feel free to correct what I got wrong)
```suggestion
    """Wrapper for `nlopt` that creates and updates a database of simulation inputs/outputs.

    Note that for large databases (or fast simulations), the database lookup can be more expensive than performing the simulation.
    """
```